content/memorials/henry_story.md

```diff
@@ -0,0 +1,30 @@
+Title:     Henry Story, September 5th 2023
+license: https://www.apache.org/licenses/LICENSE-2.0
+
+# Henry Story, 1967-07-29 - †2023-09-05
+
+Henry Story passed away, rather suddenly, in September 2023.
+
+Over the past 30 years, he made many contributions to the standards of
+the World Wide Web on a wide range of standards; from Atom to
+LinkedData to WebID and many other semantic- and big-data work.
+
+Always trying to keep the web open, fair and neutral. Labouring to
+keep the web, and its technology, available for anyone. Requiring
+as little as a smart inquisitive individual devoting time. And without
+having to enter into license agreements, pass toll-gates or becoming
+a member of some walled gardens. Latterly he was the chair of  the W3C
```

Comment:
Remove extra space between 'of' and 'the W3C': Latterly he was the chair of the W3C